lang/update_pot.sh

```diff
@@ -32,6 +32,7 @@ xgettext --default-domain="cataclysm-dda" \
          --keyword="gettext_noop" \
```

Comment:
Yes, that's why you replaced it everywhere, but why did you only *add* the new macro here and why did you not replace the old macro with the new one? Now there is still a reference to the old one here.